README.md

```diff
@@ -65,25 +121,25 @@ touch .secrets
   - Access the rails console for debugging
 
     ```bash
-    rails c
+    bundle exec rails c
     ```
```

Comment:
undo this change (if you recommend cam sh it deals with the bundle exec for you)